packages/apps/immutable-records/.eslintrc.cjs

```diff
@@ -5,6 +5,20 @@ module.exports = {
   extends: ['@kadena-dev/eslint-config/profile/next', 'next/core-web-vitals'],
   parserOptions: { tsconfigRootDir: __dirname },
   rules: {
+    // Maybe useful for libs, only causes bloat and duplication for apps
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    // This is fine for libs, but this app will not export any types
+    '@typescript-eslint/naming-convention': 'off',
+    // Works fine with typescript
+    '@rushstack/no-new-null': 'off',
+    // The official docs recommend against it: https://typescript-eslint.io/rules/typedef/
+    // Typescript is better at inferring and using `noImplicitAny` is sufficient
+    '@typescript-eslint/typedef': 'off',
     '@kadena-dev/typedef-var': 'off',
+    // This rule is too strict for our use case
+    '@typescript-eslint/strict-boolean-expressions': 'off',
+    // TODO: eslint in vscode does not understand tsconfig paths and gives errors
+    //       even though in the build it works fine
+    'import/no-unresolved': 'off',
```

Comment:
Looks like a good opportunity to have something of a shared `app` ESLint config, or extend the existing `next` config.

Those profiles are here: https://github.com/kadena-community/kadena.js/tree/main/packages/tools/eslint-config/profile

This specific `import/no-unresolved` should not need to be disabled, this works well in the other packages too?

Reply:
But I'm already extending `next` via `@kadena-dev/eslint-config/profile/next` right?
Also `next` extends `react` and `react` extends `lib`, so when I use `next` I automatically opt-in to all the `lib` rules too

Reply:
Found the solution, you have to configure `"eslint.workingDirectories": ["./packages/apps/immutable-records"]` in `./vscode/settings.json` 
Where I just have this app now, but ideally any directory that has a `.eslintrc.*` is added, really don't know why they don't just resolve these configs automatically but okay, this does work.